File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    s = list(s)
    p = 0
    y = 0
    for i in s:
        if i == 'p' or i == 'P':
            p +=1
        if i == 'y' or i == 'Y':
            y += 1
    if p == y:
        answer = True
    else:
        answer = False
    return answer

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
import java.util.*;

class Solution {
    boolean solution(String s) {
        // 대소문자 구분하지 않음.
        s = s.toLowerCase();
        int[] count = new int[]{0, 0};
        for(char x: s.toCharArray()){
            int a = ((x == 'p')?count[0]++ : ((x == 'y')?count[1]++:0));
        }
        System.out.println(Arrays.toString(count));
        boolean answer = count[0] == count[1];
        return answer;
    }
}